ncrystal_python/src/NCrystal/ncmat2endf.py

```diff
@@ -1182,23 +1193,31 @@ def ncmat2endf( ncmat_fn,
     Returns
     -------
     file_names: list of (str, float)
-        List of tuples contanining the ENDF-6 files and their fraction in the composition
+        List of tuples contanining the ENDF-6 files and
+        their fraction in the composition
 
     """
     if verbosity > 0 and endf_parameters.lasym > 0:
-        print(f'Creating non standard S(a,b) with LASYM = {endf_parameters.lasym}')
+        # TODO: replace with a warning
+        print( 'Creating non standard S(a,b)'+
+               f' with LASYM = {endf_parameters.lasym}')
 
     if type(temperatures) in [int, float]:
         temperatures = (temperatures,)
 
     if len(temperatures) > 1:
-        warnings.warn('Multiple temperatures requested. Although this is supported, '
-        +'it is not recommended because NCrystal generates a custom (alpha,beta) grid for each temperature. '
-        +'The (alpha,beta) grid for first temperature will be used, and S(alpha, beta) for other temperatures will be interpolated.', stacklevel=2)
+        warnings.warn('Multiple temperatures requested.'+
```

Comment:
Note that there is a `warn` function in the `._common` module, to be used for all warnings emitted from the NCrystal python layer.